pkg/api/core/v1/models/models.go

```diff
@@ -38,6 +38,13 @@ type ServiceCreateRequest struct {
 	Data map[string]string `json:"data"`
 }
 
+// ServiceUpdateRequest represents and contains the data needed to
+// update a service instance (add/change, and remove keys)
+type ServiceUpdateRequest struct {
+	Remove []string          `json:"remove,omitempty"`
+	Edit   map[string]string `json:"edit,omitempty"`
```

Comment:
Would `Edit` be better described as `Set` given it covers adding new entries?

Reply:
Ok. That is trivial.